### ISSUE TITLE
flexup MON?

### ISSUE THREAD

**Issue by tangzhankun** [Intel-bigdata/ceph-mesos#23]
In production, users need at least 3 mons. we can support flexup Mon. Two Mons launched at a time.

**Comment by a-nldisr** [comment]
I do not see this on the roadmap, is this a planned feature and is it planned to configure this value from 3 to 5?

**Comment by tangzhankun** [comment]
@a-nldisr 
yes. This is a must done feature if we want ceph-mesos to be production ready. I think we can let user configure a initial mon number or use a similar way like OSD flexup.
I prefer a REST API for Mon which will be '{"instances":2,"profile":"mon", "attribute":"rack1,rack2"}'. In this way, Ceph-Mesos can find two hosts in rack1 and rack2 to launch the MONs( and mesos slave should be started with "--attributes" param).
